dwave/plugins/sklearn/transformers.py

```diff
@@ -0,0 +1,257 @@
+from sklearn.base import BaseEstimator
+from sklearn.feature_selection import SelectorMixin
+from dwave.system import LeapHybridCQMSampler
+import dimod
+import numpy as np
+import pandas as pd
+import tempfile
+
+import logging
+
+class HybridFeatureSelection(BaseEstimator, SelectorMixin):
+    
```

Comment:
Needs a docstring. In Ocean we generally have a class docstring rather than an `__init__`.